proto/generate.sh

```diff
@@ -31,7 +31,7 @@ echo "* Generating Protobuf/gRPC SDK files:"
 echo -e "\tVERSION: $PROTOC_VERSION"
 echo -e "Generated by version $PROTOC_VERSION of protoc" > ./grpc_version.txt
 
-# `status.proto` is in our source tree so that we can implement initialization failure in the
+# `status.proto` are in our source tree so that we can implement initialization failure in the
```

Comment:
Yes, for sure.  This was an accident, I originally had `error_details.proto` included here, in which case it would be "these files are", but then removed it, but forgot to revert this bit.